web/views.py

```
@@ -2730,7 +2743,131 @@ def challenge_submit(request, week_number):
+        has_submitted = user_submission is not None
+
+    return render(request, "web/current_live_quiz.html", {
+        "quiz": quiz,
+        "has_submitted": has_submitted,
+        "user_submission": user_submission,
+        "questions":questions
+    })
+
+
+@login_required
+@csrf_protect
```

Ruff (0.8.2)

Undefined name `csrf_protect`

(F821)